pkg/issuer/acme/dns/dns.go

```diff
@@ -167,33 +167,33 @@ func (s *Solver) solverForIssuerProvider(providerName string) (solver, error) {
 	case providerConfig.CloudDNS != nil:
 		saSecret, err := s.secretLister.Secrets(s.resourceNamespace).Get(providerConfig.CloudDNS.ServiceAccount.Name)
 		if err != nil {
-			return nil, fmt.Errorf("error getting clouddns service account: %s", err.Error())
+			return nil, fmt.Errorf("error getting clouddns service account: %s", err)
```

Comment:
`%s` and `%v` are identical for anything that implements the `error` interface or the `Stringer` interface, and specifically both of those verbs will prefer the `Error()` method if available, and if not fall back to `String()`. Even though `%s` sounds like it should format just strings, it will go out of its way to call `Error()` / `.String()` whenever possible.

I think it's slightly nicer to not call `.Error()` when formatting, but I have no preference on '%s' or '%v' other than '%s' being mildly more specific.